CONTRIBUTING.md

```diff
@@ -15,8 +15,8 @@ Releases will be numbered with the following format:
 And constructed with the following guidelines:
 
 - Breaking backward compatibility bumps the major and resets the minor and patch.
-- New additions without breaking backward compatibility bumps the minor and resets the patch.
-- Bug fixes and misc changes bumps the patch.
+- New additions without breaking backward compatibility bump the minor and reset the patch.
+- Bug fixes and misc changes bump the patch.
```

Comment:
These lines already fixed few minutes ago in all packages ;) Thanks to your recent PR. You can remove this duplicate fix now.
https://github.com/rinvex/laravel-authy/blob/develop/CONTRIBUTING.md

Reply:
Done, thanks.